[Magic Firewall] Added best practice content (#3323)
cloudflare/cloudflare-docs: products/magic-firewall/src/content/best-practices/minimal-ruleset.md

---
title: Minimal ruleset
pcx-content-type: concept
order: 1
---

# Minimal suggested ruleset

The suggested minimal ruleset blocks some known common vectors for DDoS attacks and permits all other ESP, TCP, UDP, GRE and ICMP traffic.

> Note that this is a suggested list and not an exhaustive list. Please check what fits for your environment and add more rules as necessary.

Reviewer: ```suggestion
This is a suggested list and not an exhaustive list. Review your environment and add more rules as necessary.
```
